### PR TITLE
Update emissions for subdaos to 100% of theoretical

### PR DESCRIPTION
**Pull request by ChewingGlass**

### CODE REVIEW

**Comment by abhay** [comment]
NB: History cannot be maintained explicitly in this file due to solana transaction size limits.

File: packages/helium-admin-cli/emissions/iot.json [hunk 1]
```diff
@@ -1,18 +1,18 @@
 [
   {
-    "startTime": "2023-04-18T00:00:00Z",
-    "emissionsPerEpoch": "165,616,438.356164"
+    "startTime": "2024-01-08T00:00:01Z",
+    "emissionsPerEpoch": "88,797,814.207650"
```
Comment (abhay) [inline packages/helium-admin-cli/emissions/iot.json:4]: 88,797,814.207650*366 = 32.5B IOT yearly.

File: packages/helium-admin-cli/emissions/iot.json [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
   },
   {
-    "startTime": "2023-08-02T00:00:00Z",
-    "emissionsPerEpoch": "165,616,438.356164"
+    "startTime": "2024-08-01T00:00:01Z",
+    "emissionsPerEpoch": "89,041,095.890411"
```
Comment (abhay) [inline packages/helium-admin-cli/emissions/iot.json:8]: 89,041,095.890411*365 = 32.5B IOT annualized.

File: packages/helium-admin-cli/emissions/iot.json [hunk 3]
```diff
@@ -1,18 +1,18 @@
...
   },
   {
-    "startTime": "2023-08-03T00:00:00Z",
-    "emissionsPerEpoch": "82,581,967.213115"
+    "startTime": "2025-08-01T00:00:01Z",
+    "emissionsPerEpoch": "44,520,547.945206"
```
Comment (abhay) [inline packages/helium-admin-cli/emissions/iot.json:12]: 44,520,547.945206*365=16.25B IOT annualized

File: packages/helium-admin-cli/emissions/iot.json [hunk 4]
```diff
@@ -1,18 +1,18 @@
...
   },
   {
-    "startTime": "2024-08-01T00:00:01Z",
-    "emissionsPerEpoch": "82,808,219.178082"
+    "startTime": "2027-08-01T00:00:01Z",
+    "emissionsPerEpoch": "22,199,453.551913"
```
Comment (abhay) [inline packages/helium-admin-cli/emissions/iot.json:16]: 22,199,453.551913*366 = 8.125B IOT annualized (leap year)

File: packages/helium-admin-cli/emissions/mobile.json [hunk 5]
```diff
@@ -1,18 +1,18 @@
 [
   {
-    "startTime": "2023-04-18T00:00:00Z",
-    "emissionsPerEpoch": "108,493,150.684932"
+    "startTime": "2024-01-08T00:00:01Z",
+    "emissionsPerEpoch": "81,967,213.114754"
```
Comment (abhay) [inline packages/helium-admin-cli/emissions/mobile.json:4]: 81,967,213.114754*366 = 30B MOBILE annualized

File: packages/helium-admin-cli/emissions/mobile.json [hunk 6]
```diff
@@ -1,18 +1,18 @@
...
-    "startTime": "2023-08-02T00:00:00Z",
-    "emissionsPerEpoch": "141,369,863.013699"
+    "startTime": "2024-08-01T00:00:01Z",
+    "emissionsPerEpoch": "70,684,931.506849"
   },
   {
-    "startTime": "2023-08-03T00:00:00Z",
-    "emissionsPerEpoch": "70,491,803.278689"
+    "startTime": "2025-08-01T00:00:01Z",
+    "emissionsPerEpoch": "41,095,890.410959"
```
Comment (abhay) [inline packages/helium-admin-cli/emissions/mobile.json:12]: 41,095,890.410959*365 = 15B MOBILE annualized

File: packages/helium-admin-cli/emissions/mobile.json [hunk 7]
```diff
@@ -1,18 +1,18 @@
...
   },
   {
-    "startTime": "2024-08-01T00:00:01Z",
-    "emissionsPerEpoch": "70,684,931.506849"
+    "startTime": "2027-08-01T00:00:01Z",
+    "emissionsPerEpoch": "20,491,803.278689"
```
Comment (abhay) [inline packages/helium-admin-cli/emissions/mobile.json:16]: 20,491,803.278689*366 = 7.5B annualized

File: packages/helium-admin-cli/emissions/mobile.json [hunk 8]
```diff
@@ -1,18 +1,18 @@
...
-    "startTime": "2023-04-18T00:00:00Z",
-    "emissionsPerEpoch": "108,493,150.684932"
+    "startTime": "2024-01-08T00:00:01Z",
+    "emissionsPerEpoch": "81,967,213.114754"
   },
   {
-    "startTime": "2023-08-02T00:00:00Z",
-    "emissionsPerEpoch": "141,369,863.013699"
+    "startTime": "2024-08-01T00:00:01Z",
+    "emissionsPerEpoch": "82,191,780.821918"
```
Comment (abhay) [inline packages/helium-admin-cli/emissions/mobile.json:8]: 82,191,780.821918*365 = 30B MOBILE annualized

**Review by abhay — APPROVED** [review]

**Review by bryzettler — APPROVED** [review]

**Pull request merged**